rpc/connection.go

```diff
@@ -608,6 +623,9 @@ type connectionClient struct {
 var _ GenericClient = connectionClient{}
 
 func (c connectionClient) Call(ctx context.Context, s string, args interface{}, res interface{}) error {
+	if isWithFireNow(ctx) || c.conn.initialReconnectBackoffWindow != 0 {
```

Comment:
(The reason why it was working in my tests must be that the ticker was cancelled.)